cloud_storage_handler/main.py

```diff
@@ -39,13 +29,50 @@ def init_app() -> FlaskApp:
 
     foca = Foca(
         config_file=config_path,
+        custom_config_model="cloud_storage_handler.custom_config.CustomConfig",
     )
-    return foca.create_app()
+    foca_config = foca.conf
+    minio_config = foca_config.custom.minio
+    logger.info(minio_config)
+
```

Comment:
This should be `logger.debug(minio_config)` otherwise, in production env, minio creds will be logged.